osu.Game/Rulesets/Mods/Mod.cs

```diff
@@ -42,6 +47,48 @@ public abstract class Mod : IMod, IJsonSerializable
         [JsonIgnore]
         public virtual string Description => string.Empty;
 
+        /// <summary>
+        /// The tooltip to display for this mod when used in a <see cref="ModIcon"/>.
+        /// </summary>
+        /// <remarks>
+        /// Differs from <see cref="Name"/>, as the value of attributes (AR, CS, etc) changeable via the mod
+        /// are displayed in the tooltip.
+        /// </remarks>
+        [JsonIgnore]
+        public virtual string IconTooltip
+        {
+            get
+            {
+                List<string> attributes = new List<string>();
```

Comment:
Cases like this I'd 100% of the time use `var` to not make the code Java-level redundant.
```suggestion
                var attributes = new List<string>();
```